src/main/webapp/Update.jsp

```diff
@@ -0,0 +1,106 @@
+<%@ page import="java.sql.*" %>
+<%@ page import="main.java.com.registration.*" %>
+<%@ page import="main.java.com.registration.dbconnection.*" %>
+<% Class.forName("com.mysql.jdbc.Driver"); %>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>Update data</title>
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto|Varela+Round|Open+Sans">
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+<!-- Main css -->
+<link rel="stylesheet" href="css/style.css">
+
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+<!-- Font Icon -->
+<link rel="stylesheet" href="fonts/material-icon/css/material-design-iconic-font.min.css">
+</head>
+<body>
+<%@ include file="Navbar.jsp" %>
```

Comment:
Can you do the modification in your code in a way so we don't need to write JDBC code in the JSP file? The JSP file should only be used to display the information.

Reply:
ok sir i will do this changes